### DISCUSSION TITLE
golangci-lint run fails with: can't run linter goanalysis_metalinter: buildssa: failed to load package <package_name>: could not load export data: no export data for <package>

### DISCUSSION THREAD

**Discussion by tigrkoshka** [golangci/golangci-lint#3363] [Q&A]
I have a rather big repository with all kind of different packages, some of which use CGO. I am trying to run golangci-lint on a directory that does not use CGO, but in one files depends on a package that does, which gives me

```
can't run linter goanalysis_metalinter: buildssa: failed to load package <package name>: could not load export data: no export data for <package>
```

I have seen similar discussions, but none of the tips helped me. 

I am assuming the problem is with CGO usage, because golangci-lint reports the code that dos not compile as pure Go.

How can I enable CGO in golangci-lint run or, alternatively, how can I tell golangci-lint to ignore that package? 

If that requires ignoring my packages/files that depend on the CGO package -- that's fine, because there is only one such file. I have tried  skip-dirs or skp-files in config, but that doesn't seem to work.

I have also run `go mod tidy`, but that didn't help either.

golangci-lint version: 1.50.1
go version: 1.19

**Comment by pascal-blokur** [comment]
Same issue, and I get it while running the dockerize'd version :
` docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:v1.50.1 golangci-lint run -v`

The package that is causing the problem was generated by protoc_gen_go 1.27.1 for me.

**Comment by 3tbraden** [comment]
For me it seems like issue with golangci-lint trying to fetch the private dependencies. Assuming there's `vendor/`, add `--modules-download-mode=vendor` when running the command.

**Reply by superhorsy** [reply]
Thank you, it fixed my issue
